tests/playground.html

```diff
@@ -204,22 +204,22 @@
   }
 }
 
-function toXml() {
+function save() {
```

Comment:
I think these should probably be separate buttons instead of replacing the xml buttons, at least for now. We still use the non-advanced playground for some QA test cases and this includes exporting the XML and using the Code demo to import the XML. It could still be removed at some point in the future but I'm hesitant about removing it for now even though this is mostly a test page for us.